scene/gui/label.cpp

```diff
@@ -370,13 +420,13 @@ Size2 Label::get_minimum_size() const {
 	min_size.height = MAX(min_size.height, font->get_height(get_theme_font_size("font_size")) + font->get_spacing(Font::SPACING_TOP) + font->get_spacing(Font::SPACING_BOTTOM));
 
 	Size2 min_style = get_theme_stylebox("normal")->get_minimum_size();
-	if (autowrap) {
-		return Size2(1, clip ? 1 : min_size.height) + min_style;
+	if (autowrap_mode != AUTOWRAP_OFF) {
+		return Size2(1, (clip || overrun_behavior != OVERRUN_NO_TRIMMING) ? 1 : min_size.height) + min_style;
 	} else {
-		if (clip) {
+		if (clip || overrun_behavior != OVERRUN_NO_TRIMMING) {
 			min_size.width = 1;
 		}
-
+		auto res = min_size + min_style;
```

Comment:
Using `auto` is usually against Godot code style.

Reply:
Oops, that line was for debugging and I forgot to remove it. Fixed.